src/Main.java

```diff
@@ -4,14 +4,23 @@
 import command.RegisterCommand;
 import command.constant.CommandConstants;
 import container.UserContainer;
+import file.UserFromFileReader;
+import model.User;
 
+import java.util.List;
 import java.util.Scanner;
 
 public class Main {
-
     private static final Scanner scanner = new Scanner(System.in);
 
+    public void init() {
```

Comment:
init method should init application
it means it should init container
it means it should read users form file and save it to container